### PR TITLE
Show test run and output correctly

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/ruby/issues/1200.

@iHiD It is an escaping issue.

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]
@kntsoriano Is the `messageHtml` field properly named then? I would expect HTML to be rendered dangerously, but as the bug is fixed by not escaping it, is the value really HTML?

**Comment by iHiD** [comment]
Yeah, I'm not sure this is correct. `messageHtml` **is** HTML, so this is going to break as soon as there's ansi used in the message I think. @kntsoriano I think the issue is more that things need escaping in `app/commands/ansi/render_html.rb`

**Comment by iHiD** [comment]
Good catch, @ErikSchierboom.

Replacing this with https://github.com/exercism/website/pull/1890.

**Comment by iHiD** [comment]
Nice. @joshgoebel Better?

**Comment by iHiD** [comment]
This is green for me locally 🤷

File: app/javascript/components/editor/TestSummary.tsx
Comment (joshgoebel) [inline app/javascript/components/editor/TestSummary.tsx:63]: This still seems problematic... what is the content of `test.messageHtml` exactly in this case?  Is there a non-zero chance that it can contain ANY user generated content at all (custom test errors, etc)?  Is any such content  guaranteed to be pre-escaped by the test runner itself?

Comment (joshgoebel) [reply app/javascript/components/editor/TestSummary.tsx:63]: So far in my experience `dangerouslySetInnerHTML` should require like the sign-off of 5 different people before it's approved. :-)

Comment (joshgoebel) [reply app/javascript/components/editor/TestSummary.tsx:63]: Allowing test runners (which run user code!) to generate HTML at all seems problematic to me from a security standpoint. _Test runners should generate JSON data that the website can interpret in a secure fashion, not raw HTML._

Comment (iHiD) [reply app/javascript/components/editor/TestSummary.tsx:63]: > Test runners should generate JSON data that the website can interpret in a secure fashion, not raw HTML.

They do.

Comment (iHiD) [reply app/javascript/components/editor/TestSummary.tsx:63]: However, they are preprocessed before being displayed (e.g. they ansi is converted to colours)

Comment (joshgoebel) [reply app/javascript/components/editor/TestSummary.tsx:63]: That's fine. But RAW html from test runners should probably not be allowed/escaped.

**Comment by iHiD** [comment]
@kntsoriano This is still red.

**Comment by iHiD** [comment]
I fixed it :)

**Pull request merged**
